data/sql/updates/pending_db_world/rev_1676663461812301700.sql

```diff
@@ -0,0 +1,8 @@
+-- Infected Kodo Beast - add missing ability, Stampede.
+DELETE FROM `creature_template_spell` WHERE `CreatureID`=25596;
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+(25596, 0, 45876, 12340),
+(25596, 1, 45877, 12340);
+
+-- Column Ornament - disable gravity, previously falling to the ground.
+UPDATE `creature_template_movement` SET `Flight` = 1 WHERE `Flight` = 0 AND `CreatureId` = 29754;
```

Comment:
this part isn't needed 😛 
```suggestion
UPDATE `creature_template_movement` SET `Flight` = 1 WHERE `CreatureId` = 29754;
```